package.json

```diff
@@ -52,6 +52,7 @@
     "husky": "8.0.1",
     "jest": "28.1.3",
     "jest-extended": "3.1.0",
+    "jest-junit": "^14.0.1",
```

Comment:
Version number needs to be fixed I believe. i.e. remove the `^` :)

Reply:
Yes, we use fixed versions everywhere, to avoid unexpected upgrades

Reply:
Done.